src/components/PostsList.tsx

```diff
@@ -1,86 +1,66 @@
 import React from 'react';
+import { Post } from '../types/Post';
+import cn from 'classnames';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  posts: Post[];
+  currPost: Post | null;
+  setOpenModal: (value: boolean) => void;
+  setCurrPost: (postId: Post | null) => void;
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  posts,
+  currPost,
+  setOpenModal = () => {},
```

Comment:
Setting a default value for `setOpenModal` can lead to unexpected behavior if the parent component does not pass this prop explicitly. Consider removing the default value to ensure that the parent component always provides this function.